ena-submission/scripts/ena_submission_helper.py

```diff
@@ -129,6 +129,18 @@ def get_project_xml(project_set):
         "PROJECT": dataclass_to_xml(project_set, root_name="PROJECT_SET"),
     }
 
+def reformat_authors(authors: str) -> str:
```

Comment:
```suggestion
def reformat_authors_from_loculus_to_embl_style(authors: str) -> str:
```

Maybe something like this is a bit more explicit - reformat could mean anything